documentation/multinode-systems.html

```diff
@@ -0,0 +1,63 @@
+---
+# Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+title: "Multinode systems"
+---
+
+<p>A Vespa <i>system</i> consists of one or more stateless and stateful clusters configured by one application package.
+A Vespa system is configured and managed through an admin cluster as shown below.</p>
+
+<img src="img/overview/vespa-overview.png"/>
+
+<p>All nodes of a Vespa system has the same software installed. Which processes are started on each node and
+how they are configured is determined by the admin cluster from the specification given in the 
+services.xml file in the application package.</p>
+
+
+<h2 id="making-changes-to-live-systems">Making changes to live systems</h2>
+
+<p>To change the system, deploy the changed application to the admin cluster.
+The admin cluster will automatically change the participating nodes as necessary.
+It is safe to do this while serving live query and write traffic. 
+In rare cases the admin cluster will tell you that some processes must be restarted
+to make the change effective. To avoid query or write traffic disruption, such
+restarts must be done on one node at the time,
+and you must wait until the node is fully up before restarting the next node.</p>
+
+
+<h2 id="creating-a-multinode-system-from-a-sample-application">Creating a multinode system from a sample application</h2>
+
+<p>To create a fully functional production ready multinode system from a single-node sample application,
+follow these steps:
+
+<ul>
+<li>Add an explicit admin cluster to your services.xml file:
+<pre>
+&lt;services version=&quot;1.0&quot;&gt;
```

Comment:
oh I dislike the &amp;quot;  can you not just use  regular " ? ;-)

Reply:
I agree with you. I just use my editor to convert to entities automatically. I'll try to avoid them in the future.